### PR TITLE
Add Linkerd as a project

### PR DESCRIPTION
**Pull request by grampelberg**

### CODE REVIEW

File: _data/projects/linkerd.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- javascript
+- kubernetes
+- rust
+- CNCF
+- containers
+- docker
+- react
+upforgrabs:
+  name: good first issue
+  link: https://github.com/linkerd/linkerd2/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```
Comment (shiftkey) [inline _data/projects/linkerd.yml]: :art: this is a cleaner URL that will give you the same result:

```suggestion
  link: https://github.com/linkerd/linkerd2/labels/good%20first%20issue
```

**Review by shiftkey — COMMENTED** [review]
@grampelberg looking good, just one suggestion to tidy up

File: _data/projects/linkerd.yml (hunk 1 above)
Comment (grampelberg) [reply _data/projects/linkerd.yml]: Cool! I didn't realize that only selected open issues =)

**Comment by grampelberg** [comment]
Updated!

**Review by shiftkey — APPROVED** [review]

**Pull request merged**